OpenRobertaServer/staticResources/index.html

```diff
@@ -1122,7 +1122,26 @@ <h3 class="modal-title" lkey="Blockly.Msg.POPUP_ATTENTION">Attention</h3>
                 </div>
             </div>
         </div>
-    </div>
+    </div>   
```

Comment:
Is a whole other popup actually necessary? I am not sure. What about changing the message to show the robot and using the default "confirm and cancel" popup to either switch or not. You can look up other uses of this workflow (e.g. https://github.com/OpenRoberta/openroberta-lab/blob/develop/OpenRobertaServer/staticResources/js/app/roberta/controller/menu.controller.js#L631).